configs/templates/hpc-stack-dev/spack.yaml

```diff
@@ -1,4 +1,7 @@
 spack:
+  specs:
```

Comment:
This will be ignored, because `specs` is coming further down in the list. Let me try to fix this on hera and send you an update.